src/resizable-card-layout.scss

```diff
@@ -0,0 +1,132 @@
+@import './new-settings';
+@import './mixins';
+@import './icon';
+
+$block: #{$fd-namespace}-resizable-card;
+
+$fd-resize-handle-width: 0.0625rem;
+$fd-border-on-drag: 0.125rem dashed var(--sapContent_DragAndDropActiveColor);
+
+.#{$block} {
```

Comment:
There also should be added container, that adds some padding per sides for cards.